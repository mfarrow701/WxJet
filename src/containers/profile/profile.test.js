import React from 'react';
import {shallow} from 'enzyme';
import Profile from './profile';

describe('Profile container', () => {
    let component;

    const renderWrapper = () => {
        component = shallow(<Profile/>);
    };

    beforeEach(() => {
        renderWrapper();
    });

    it('should render the profile container after load', () => {
        expect(component).toMatchSnapshot();
        expect(component.find('.Profile h1').text()).toEqual('Profile');
    });
});
