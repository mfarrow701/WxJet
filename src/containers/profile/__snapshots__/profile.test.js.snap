// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Profile container should render the profile container after load 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Profile />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <h1>
        Profile
</h1>,
      "className": "Profile",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": "Profile",
      },
      "ref": null,
      "rendered": "Profile",
      "type": "h1",
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <h1>
          Profile
</h1>,
        "className": "Profile",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Profile",
        },
        "ref": null,
        "rendered": "Profile",
        "type": "h1",
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
